docs/accessing_data.md

```diff
@@ -134,6 +134,39 @@ be used directly from the Treeherder repository [here][gcp-cert] for the stage r
 [gcp-cert]: https://github.com/mozilla/treeherder/blob/master/deployment/gcp/ca-cert.pem
 [gcp-prototype-cert]: https://github.com/mozilla/treeherder/blob/master/deployment/gcp/ca-cert-prototype.pem
 
+You can alternatively connect to a staging database with a read-only proxy server. To do this follow these for credentials and treeherder setup:
+
+1. [File a Jira ticket here](https://mozilla-hub.atlassian.net/jira/software/c/projects/SVCSE/boards/316) to get
+credentials to access the databases, give the ticket a title of: "Request credentials for Treeherder's Staging database"
+2. Once the credentials are provided, edit the .env to contain: DATABASE_URL=psql://username:password@host.docker.internal:5432/treeherder
+3. Edit docker/entrypoint.sh to contain:
+
+```shell
+# Keep these in sync with DATABASE_URL.
+echo "Checking database status at $DATABASE_URL"
+if [[ ${DATABASE_URL:0:8} == "mysql://" ]]; then
+ check_service "MySQL" "mysql" 3306;
+fi
+if [[ ${DATABASE_URL:0:27} == *"@host.docker.internal"* ]]; then
+ check_service "PostgreSQL" "host.docker.internal" 5432;
+elseNinN
+ check_service "PostgreSQL" "postgres" 5432;
+fi
+```
+
+Then, in your local Environment run the following:
+
+1. [Download the Cloud SQL Auth Proxy](https://cloud.google.com/sql/docs/postgres/connect-auth-proxy#install)
+2. [Follow the steps in the Treeherder Dev Resources](https://mozilla-hub.atlassian.net/wiki/spaces/SRE/pages/671514712/Treeherder+Dev+Resources#Database-Instance-Names) the steps being:
+   1) Run `gcloud auth login --update-adc`
+   2) Run `./cloud-sql-proxy --address 127.0.0.1 --port 5432 moz-fx-treeherde-nonprod-34ec:us-west1:treeherder-nonprod-stage-v1-postgres-replica-0`
+
+Notes:
+
+- If you’re having issues on linux related to endpoint postgres, and/or bind: address already in use you can try changing the port from 5432 to 5433
```

Comment:
Based on my local documentation, I ended up changing the postgres container port to 5433. Then, I kept using the 5432 port in the commands above.